src/views/Shapes/Capsule.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Modifiers } from '../../utils/modifiers';
+import { Rectangle } from './Rectangle';
+
+type CapsuleProps = Omit<Modifiers, 'backgroundColor'> & {
+  fill: string;
+  frame: { width?: number; height?: number };
```

Comment:
should we do

frame: { width: number; } | { height: number } | { width: number; height: number };

Reply:
fill is required on Capsule but not the others it looks like, did we mean to make it optional?